roles/asset_pipeline/tasks/main.yml

```diff
@@ -1,6 +0,0 @@
-- name: Precompile assets
```

Comment:
I wrote this role a while ago, but I don't think it was ever used -- it's not listed in `omnibox.yml`. The tasks were just rolled into the `database` role, so I'm deleting them here.